google/cloud/storage/internal/resumable_upload_session.h

```diff
@@ -40,6 +40,17 @@ class ResumableUploadSession {
    * @return The result of uploading the chunk.
    */
   virtual StatusOr<ResumableUploadResponse> UploadChunk(
+      std::string const& buffer) = 0;
+
+  /**
+   * Uploads a chunk and returns the resulting response.
```

Comment:
Does `UploadFinalChunk()` deserve a different description to `UploadChunk()`?

Reply:
Fixed.